src/main/java/graphql/execution/FieldCollectorParameters.java

```diff
@@ -37,12 +37,6 @@ private FieldCollectorParameters(GraphQLSchema graphQLSchema, Map<String, Object
         this.objectType = objectType;
     }
 
-    public static Builder newParameters(GraphQLSchema graphQLSchema, GraphQLObjectType objectType) {
```

Comment:
Would you like me to revert this change entirely and keep the method signature the same?

Or I can change it to

```
public static Builder newParameters() {
    return new Builder();
}
```

with the assertions moved to `public FieldCollectorParameters build() {...}`, as they are in this PR.